src/pages/index.js

```diff
@@ -141,6 +147,23 @@ const SOLUTIONS = {
       description: 'Spotify experienced an explosion of docs writing after adopting this methodology internally.',
       icon: PresentationChartLineIcon,
     }],
+  }, {
+    title: 'Track engineering maturity and migrations',
+    description: 'Define engineering standards and initiatives that get checked automatically across your ecosystem.',
+    prompt: 'Coming soon',
+    bullets: [{
+      name: 'A Roadie Exclusive Feature',
+      description: 'Standards and initiatives tracking is not available to self-hosted Backstage users.',
+      icon: SparklesIcon,
```

Comment:
We should leave this one out. I'm not 100% sure this is true yet tbh. We'll have to see.